quarkchain/constants.py

```diff
@@ -35,16 +35,16 @@
 
 # The following code is compiled with:
 #    solc (0.5.17+commit.d19bba13.Linux.g++) --bin --optimize --optimize-runs 200 --evm-version petersburg
-# Source code commit 896b2f38.
+# Source code commit da899a29.
 NON_RESERVED_NATIVE_TOKEN_CONTRACT_BYTECODE = bytes.fromhex(
     """
```

Comment:
thought you are going to use a different address for supervisor?

Reply:
Will change to another address later.